app/views/lead_providers/content/partnership_guide.md

```diff
@@ -0,0 +1,188 @@
+We’ve created an online service for training providers to:
+
+- upload the schools they have recruited
+- check which schools have added their early career teachers and mentors
+
+Our service should help you through each step but here is a quick guide to get you up to speed.
+
+##Contents
```

Comment:
This isn't rendering correctly in the rich-diff because of the lack of space after the hashes

Reply:
Added spacing to fix renderer issues

Reply:
Added spacing to fix renderer issues